### PR TITLE
CRIO config for OVN-K CNI plugin

### PR DESCRIPTION
**Pull request by oglok**
**Which issue(s) this PR addresses**:

Closes [USHIFT-143](https://issues.redhat.com/browse/USHIFT-143) and [USHIFT-144](https://issues.redhat.com/browse/USHIFT-144)

### CODE REVIEW

File: packaging/rpm/microshift.spec [hunk 1]
```diff
@@ -255,9 +257,14 @@ sed -i -n -e '/^OPTIONS=/!p' -e '$aOPTIONS="--no-mlockall"' /etc/sysconfig/openv
 %{_bindir}/configure-ovs-microshift.sh
 
 %changelog
+<<<<<<< HEAD
```
Comment (zshi-redhat) [inline packaging/rpm/microshift.spec:260]: resolve conflict :-)

File: packaging/rpm/microshift.spec [hunk 2]
```diff
@@ -157,6 +157,7 @@ restorecon -v %{buildroot}%{_bindir}/microshift
 
 install -d -m755 %{buildroot}%{_sysconfdir}/crio/crio.conf.d
 install -p -m644 packaging/crio.conf.d/microshift.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/microshift.conf
+install -p -m644 packaging/crio.conf.d/microshift-ovn.conf %{buildroot}%{_sysconfdir}/crio/crio.conf.d/microshift-ovn.conf
```
Comment (zshi-redhat) [inline packaging/rpm/microshift.spec:160]: In the case there are two microshift*.conf installed, which one takes preference?

Comment (oglok) [reply packaging/rpm/microshift.spec:160]: I think CRI-O can take several config files. It's just a way to arrange configs.

Comment (mangelajo) [reply packaging/rpm/microshift.spec:160]: Correct, alphabetical order, all options "added"

**Pull request merged**